templates/debug.twig

```diff
@@ -69,18 +94,34 @@
             </div>
         </nav>
 
+        <svg aria-hidden="true" style="position: absolute; width: 0; height: 0; overflow: hidden;" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+            <defs>
+                <symbol id="tm-icon-checkmark" viewBox="0 0 32 32">
+                    <path d="M27 4l-15 15-7-7-5 5 12 12 20-20z"></path>
+                </symbol>
+            </defs>
+        </svg>
+
         <div class="page" style="clear:both;">
             <div id="secondNavBar" class="Menu--dashboard z-depth-1"
                  ng-show="mtmDebug.contentTab != 'logs'">
                 <ul class="navbar" role="menu" style="padding: 0;">
                     <li class="menuTab" role="menuitem">
-                        <span class="item" style="font-weight: normal;"> Events</span>
+                        <span class="item" style="font-weight: normal;"> Events
+                            <span ng-hide="(mtmDebug.mtmEvents|length) == 0">
+                                <br><br>
+                                <input type="checkbox" class="onlyFiredTags-chk" name="onlyfiredTags" id="onlyfiredTags" value="1" ng-model="onlyfiredTags">
+                                <label for="onlyfiredTags" class="lbl-onlyfiredTags">Only fired tags</label>
+                            </span>
+                        </span>
                     </li>
-                    <li ng-show="(mtmDebug.mtmEvents|length) == 0">No event executed</li>
+                    <li ng-show="(mtmDebug.mtmEvents|length) == 0" style="padding: 0 0 1rem 1.2rem;">No event executed</li>
                     <li class="menuTab" role="menuitem" ng-class="{'active': eventIndex === mtmDebug.selectedEventIndex}"
-                    ng-repeat="event in mtmDebug.mtmEvents | orderBy:'$index':true">
+                    ng-repeat="event in mtmDebug.mtmEvents | orderBy:'$index':true" ng-hide="!event.tags.length && onlyfiredTags">
                         <a class="item ng-binding"  ng-click="mtmDebug.selectEvent(event.index0)" title="Time: {{ '{{ event.time }}'|raw }}. Trigger: '{{ '{{ event.metTrigger.name }}'|raw }}'">
-                            {{ '{{ event.index }}'|raw }}: {{ '{{ event.name }}'|raw }}
+                            {{ '{{ event.index }}'|raw }}: {{ '{{ event.name }}'|raw }} <span title="Events with fired tags" ng-show="event.tags.length">
```

Comment:
can we rename the title to `This tag was fired` ?

Reply:
Updated